Templates/SendGrid-CSharp/function.json

```diff
@@ -10,6 +10,7 @@
             "type": "sendGrid",
             "name": "message",
             "direction": "out",
+            "apiKey":  "SendGridApiKey",
             "from": "Azure Functions <samples@functions.com>",
```

Comment:
Wait, these `userPrompt`s shouldn't have `connection` and `queueName` for SendGrid, right? https://github.com/Azure/azure-webjobs-sdk-templates/blob/dev/Templates/SendGrid-CSharp/metadata.json#L11-L12. Maybe that was a copy/paste error from a previous release?